examples/RESTBase/Monitoring.yaml

```diff
@@ -0,0 +1,20 @@
+# Example similar to: https://github.com/wikimedia/restbase/blob/master/v1/content.yaml#L284
+#
+# Example run: ./phester.php test https://en.wikipedia.org/api/rest_v1/ examples/RESTBase/Monitoring.yaml
+
+suite: RESTBase monitoring page/html/{title}
+description: Test Monitoring on page/html/{title}
+type: REST
+
+tests:
+  - description: Get html by title from storage
```

Comment:
How is this different from the request that transparently generates the HTML?
And how do we know that in this case, the rendered page is already in the cache?

Reply:
Testing wise not very different. This is just one of the monitoring examples that service checker is running: https://github.com/wikimedia/restbase/blob/master/v1/content.yaml#L284

Reply:
It is pretty much the same thing in phester sence, yes. I apologise for pointing @clarakosi to such similar tests.

We know that it's in storage cause it's a production check test, it excutes continuously over the same page and after the first execution the page is guaranteed to be in storage.

Reply:
I went ahead and removed it since testing wise its pretty similar to the other one.